### PR TITLE
enable onnx 1.12 to run

### PR DESCRIPTION
**Pull request by causten**
removed the -g1 optimization due to compiler bug.

### CODE REVIEW

**Review by pfultz2 — APPROVED** [review]

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -347,8 +349,8 @@ jobs:
         rbuild build -d cget -s gh -T check \
           -DCMAKE_BUILD_TYPE=${{matrix.configuration}} \
           -DMIGRAPHX_ENABLE_PYTHON=${{matrix.configuration == 'release' && 'On' || 'Off'}} \
-          -DCMAKE_CXX_FLAGS_DEBUG="-g1 -Os -fdebug-prefix-map=$PWD=. -fdebug-types-section -fno-omit-frame-pointer -fsanitize=undefined -fno-sanitize-recover=undefined" \
-          -DCMAKE_CXX_FLAGS_CODECOV="-g1 -Og -fdebug-prefix-map=$PWD=. -fdebug-types-section -fprofile-arcs -ftest-coverage -fno-omit-frame-pointer" \
+          -DCMAKE_CXX_FLAGS_DEBUG="-g0 -Os -fdebug-prefix-map=$PWD=. -fdebug-types-section -fno-omit-frame-pointer -fsanitize=undefined -fno-sanitize-recover=undefined" \
+          -DCMAKE_CXX_FLAGS_CODECOV="-g0 -Og -fdebug-prefix-map=$PWD=. -fdebug-types-section -fprofile-arcs -ftest-coverage -fno-omit-frame-pointer" \
```
Comment (TedThemistokleous) [inline .github/workflows/ci.yaml:353]: @pfultz2  Do we want to remove debug entirely? -g0 sets it to no debug flags at all vs simple backtraces with g1

**Comment by TedThemistokleous** [comment]
@causten any more changes for this?

**Comment by causten** [comment]
yes, still need to get past compiler hangs

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (pfultz2) [reply .github/workflows/ci.yaml:353]: Yea debug symbols are generally useful for debug traces. I didnt realize this disabled debug symbols. This was mainly to workaround compiler hangs. Maybe we could just conditionally run it in 22.04.

**Comment by pfultz2** [comment]
> yes, still need to get past compiler hangs

If hangs only happen on 22.04 debug, then we could just disable that step. We still run the debug version with 20.04.

**Review by TedThemistokleous — APPROVED** [review]

**Pull request merged**